### PR TITLE
adds windows support

### PR DESCRIPTION
**Pull request by slofurno**
changes the delimiter back to its original

### CODE REVIEW

File: dnvm.sh [hunk 1]
```diff
@@ -848,10 +858,11 @@ dnvm()
             [[ ! -d $_DNVM_USER_PACKAGES ]] && echo "$_DNVM_RUNTIME_FRIENDLY_NAME is not installed." && return 1
 
             local searchGlob="$_DNVM_RUNTIME_PACKAGE_NAME-*"
-
             local runtimes=""
             for location in `echo $DNX_HOME | tr ":" "\n"`; do
                 location+="/runtimes"
+				
+				echo $location
```
Comment (jasonwilliams200OK) [inline dnvm.sh]: Nit: use of tabs instead of spaces for indentation.

File: dnvm.sh [hunk 2]
```diff
@@ -64,15 +80,7 @@ _DNVM_DNVM_DIR="$DNX_USER_HOME/dnvm"
...
-__dnvm_current_os()
-{
-    local uname=$(uname)
-    if [[ $uname == "Darwin" ]]; then
-        echo "darwin"
-    else
-        echo "linux"
-    fi
-}
+
```
Comment (jasonwilliams200OK) [inline dnvm.sh]: Nit: unrequired linefeed.

File: dnvm.sh [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # dnvm.sh
 # Source this file from your .bash-profile or script to use
 
+
```
Comment (jasonwilliams200OK) [inline dnvm.sh]: Nit: unrequired linefeed.

**Comment by jasonwilliams200OK** [comment]
:bulb: You may want squash three commits into one.

**Comment by pranavkm** [comment]
@slofurno this PR needs to go to https://github.com/aspnet/dnvm

**Comment by slofurno** [comment]
thanks for the feedback i've made the suggested changes and resubmitted at https://github.com/aspnet/dnvm/pull/451

**Pull request closed without merging**